sklearn/tests/test_public_functions.py

```diff
@@ -148,6 +148,7 @@ def _check_function_param_validation(
     "sklearn.model_selection.train_test_split",
     "sklearn.random_projection.johnson_lindenstrauss_min_dim",
     "sklearn.svm.l1_min_c",
+    "sklearn.feature_selection.r_regression",
```

Comment:
Please move it to keep alphabetical order

Reply:
> Please move it to keep alphabetical order

Line moved to keep the alphabetical ordering